README.md

```diff
@@ -322,6 +360,32 @@ A style guide for the [HTML Template Language](https://docs.adobe.com/docs/en/ht
       <sly data-sly-call="${teaserTemplates.teaserSmall @ teaserModel=teaser}"></sly>
     </sly>
     ```
+
+  - [4.11](#4.11) <a name='4.11'></a> **Avoid unnecessary `<sly>` tags.**
+  
+    It's cleaner and easier to understand your intentions if you add your block statements in the relevant elements directly instead of wrapping them with an `sly` tag.
+
+    ```html
+    <!--/* Bad */-->
+    <sly data-sly-test.title="${component.title}">
+      <h1>${title}</h1>
+    </sly>
+    
+    <!--/* Good */-->
+    <h1 data-sly-test.title="${component.title}">${title}</h1>
+    ```
+    
+  - [4.12](#4.12) <a name='4.12'></a> **Use an explicit `</sly>` end tag to close `<sly>` tags.**
+  
+    Beacuse `sly` is neither a void nor a foreign element (See [html5 start tags](https://html.spec.whatwg.org/multipage/syntax.html#start-tags)), it must be explicitly closed with and end tag `</sly>`. Using a self-closing tag is **not** allowed.
```

Comment:
```suggestion
    Because `sly` is neither a void nor a foreign element (See [html5 start tags](https://html.spec.whatwg.org/multipage/syntax.html#start-tags)), it must be explicitly closed with and end tag `</sly>`. Using a self-closing tag is **not** allowed.
```